src/main/java/com/salesforce/storm/spout/dynamic/persistence/ZookeeperPersistenceAdapter.java

```diff
@@ -89,7 +89,8 @@ public void open(Map spoutConfig) {
 
         this.curator = CuratorFactory.createNewCuratorInstance(
             // Take out spout persistence config and strip the key from it for our factory.
-            Tools.stripKeyPrefix("spout.persistence.zookeeper.", spoutConfig)
+            Tools.stripKeyPrefix("spout.persistence.zookeeper.", spoutConfig),
+            "PersistenceAdapter"
```

Comment:
Use this.class.getSimpleName()

Reply:
But that'll be the same regardless if its the Sideline one or the DynamicSpout one because they have the same class name.

Reply:
Ah, interesting...yeah I mean I guess we could prefix it before the class. Or find a way to abbreviate the package to one or two characters and do the FQN. I just prefer precision and typing on this sort of stuff.